devices/models/fan.py

```diff
@@ -18,8 +18,8 @@ def get_method(self):
                 'rotacja': self.rotation}
         return method
 
-    def __call__(self,method,power= None):
-        return self.method[method](power)
+    def __call__(self,chose_method):

```

Comment:
select_method? use_method? option? device_option? Method is vague, as I noted earlier. Also,  after comma comes the space...